extend.php

```diff
@@ -1,16 +1,33 @@
 <?php
 
+/*
+ * This file is part of fof/nightmode.
+ *
+ * Copyright (c) 2020 FriendsOfFlarum.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
 namespace FoF\NightMode;
 
 use Flarum\Extend;
+use FoF\Components\Extend\AddFofComponents;
+use FoF\Extend\Extend as FoFExtend;
```

Comment:
@davwheat sorry, my comments are getting mixed up due to something going on with my connection.

What I meant to say is that use FoF\Extend\Extend as FoFExtend; doesn't point or reference to any existing class in your code so my IDE is throwing warnings at me about it.